maps/EventMaps/mapevent.dm

```diff
@@ -0,0 +1,56 @@
+proc/LoadEventMap()
+
+	var/list/potentialEventMap = list()
+	world << "\red \b Searching for Event Map..."
+	var/list/Lines = file2list("maps/EventMaps/fileList.txt")
+	if(!Lines.len)	return
+	for (var/t in Lines)
```

Comment:
mm, that was just a quick hack to have it working for the event. I gotta add it so admins can manually choose the specific map, unless you want to do that while you're in here.